### PR TITLE
ObjectText: factored-out text related methods from TextRender 

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: src/main/perl/ObjectText.pm [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+    }
+
+And use it via
+    my $sc = SubClass->new(log => $self);
+    print "$sc"; # stringification
+
+    $sc = SubClass->new(log => $self);
+    # return CAF::FileWriter instance (text already added)
+    my $fh = $sc->filewriter('/some/path');
+    if (!defined($fh)) {
```
Comment (stdweird) [inline src/main/perl/ObjectText.pm]: @ned21 what sort of error handling do you want to see here? simple failure or also catching `thorw_error`?

**Comment by stdweird** [comment]
@ned21 ping?

File: src/main/perl/ObjectText.pm (hunk 1 above)
Comment (ned21) [inline src/main/perl/ObjectText.pm]: I am not sure I understand the question sorry. What's the difference between the two approaches?

Comment (stdweird) [inline src/main/perl/ObjectText.pm]: creating/closing filewriter instance could throw an error, which is not catched in this example. but this code does not throw any errors itself, so as an example of how to use this module, this is sufficient.

**Comment by jrha** [comment]
@ned21 any update?

**Comment by jrha** [comment]
There is a whole chain of PRs based on this one, so I'm minded to merge it if there are no real concerns with it.

**Pull request merged**